src/SFA.DAS.Approvals.Api/Models/GetStandardResponse.cs

```diff
@@ -2,35 +2,53 @@
 using System.Collections.Generic;
 using System.Linq;
 using SFA.DAS.Approvals.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.Models;
 
 namespace SFA.DAS.Approvals.Api.Models
 {
     public class GetStandardResponse
     {
-        public int Id { get; set; }
+        public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
+        public int LarsCode { get; set; }
+        public string Status { get; set; }
         public string Title { get; set; }
-        public int Level { get ; set ; }
+        public int Level { get; set; }
+        public string Version { get; set; }
```

Comment:
This refers to multiple classes, where you have added a string Version and interger VersionMajor and VersionMinor, properties, shouldn't the string Version property be a calculated property based on VersionMajor and VersionMinor?

Reply:
Given the data from Ifate comes in as a string, it seemed best to store that as is so we hold the original value. The VersionMajor VersionMinor is the calculated aspect from splitting the original field. The sole purpose of the split was purely for ordering purposes at the database level. It is a little bit duplicated but it seemed the simplest option.